test/embulk/input/test_mixpanel.rb

```diff
@@ -46,9 +47,13 @@ def setup_client
         end
       end
       def satisfy_task_ignore_start_time(expected_task)
-        satisfy{|input_task|
+        satisfy {|input_task|
           assert_not_nil(input_task[:job_start_time])
-          assert_equal(expected_task, input_task.merge(job_start_time: expected_task[:job_start_time]))
+          # This will also verify incremental column upper limit
```

Comment:
Should be nice to have some comments to explain what we're doing here, for ex:
```
# This will match `input_task` with `expected_task`, except for `:job_start_time`, as it's automatically added during runtime
```